packages/legacy/app/ios/ariesbifold/Info.plist

```diff
@@ -42,7 +42,7 @@
 	<key>NSFaceIDUsageDescription</key>
 	<string>$(PRODUCT_NAME) Wants to use your FaceID/TouchID to Authenticate your Identity</string>
 	<key>NSLocationWhenInUseUsageDescription</key>
-	<string></string>
+	<string>Our app does not request this permission or utilize this functionality but it is included in our Info.plist since our app utilizes the react-native-permissions library, which references this permission in its code.</string>
```

Comment:
Kind of technical for a human facing explanation or is it just because its Bifold and meant to be changed?
```
We include this permission because the 3rd party library library references it, even though our app doesn't use it.
```